package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
4 spaces instead of 2 to match the other .json files in the root of the project?

Reply:
I've updated this to use the 4 space indentation. In the future, this may change again if someone uses NPM rather than Yarn to manage this file. NPM will reset the file to two spaces but Yarn will respect the current indentation.